benchmarks/benchmark_low_bit_adam.py

```diff
@@ -98,6 +98,7 @@ def get_parser():
     parser.add_argument("--run_name", default="debug")
     parser.add_argument("--profile", action="store_true")
     parser.add_argument("--seed", type=int)
+    parser.add_argument("--device", type=str, choices=["cuda", "xpu"], default="cuda")
```

Comment:
Can we auto-detect device instead? For a given PyTorch installation, only either one of cuda or xpu can be available  right? It's not possible to have both cuda and xpu PyTorch at the same time?